### ISSUE TITLE
requireNativeComponent: "BVLinearGradient" was not found in the UIManager when I use Expo

### ISSUE THREAD

**Issue by sylainx** [chramos/react-native-skeleton-placeholder#75]
I've been trying to implement se Package in my **Expo** project for a few days. Without success. I keep getting the same error

` 
"dependencies": {

    "react-native-paper": "4.9.2",

    "@expo/vector-icons": "^12.0.0",

    "expo-constants": "~12.1.3",

    "react-native-skeleton-placeholder": "*",

    "@react-native-masked-view/masked-view": "0.2.5",

    "react-native-linear-gradient": "2.5.6" 
}
`

**Comment by williammulianto** [comment]
@sylainx Hi, did you manage to solve this?

**Comment by alexander01202** [comment]
No one solved this yet??

**Comment by Equilibris** [comment]
I'm also having this issue

**Comment by chramos** [comment]
hey guys, this package is not compatible with expo yet

**Comment by zacker101** [comment]
> hey guys, this package is not compatible with expo yet

When will this be available for expo? please configure it asap

**Comment by SurendraVSS** [comment]
Please configure?

**Comment by Je4stine** [comment]
No yet solved?

**Comment by SurendraVSS** [comment]
Yes, you need to go for an alternative package bcoz it does not work with
expo or you can use react cli. This package works good with react cli.

**Comment by djw27** [comment]
See https://github.com/chramos/react-native-skeleton-placeholder/issues/65#issuecomment-1311809097 for a potential workaround

**Comment by NabeelUppel** [comment]
I believe you have to use the dev-client.
https://docs.expo.dev/development/create-development-builds/

This creates a client that allows BVLinearGradient to be registered

**Comment by YasserShkeir** [comment]
Check the discussion for Expo Fix:

https://github.com/chramos/react-native-skeleton-placeholder/discussions/114

**Comment by dai-ichy** [comment]
@YasserShkeir thanks for the effort on the fix, but reaching into a package and manually applying patches isn't the way to go.

If there are no plans to provide the fix, at least the documentation can mention it.

**Comment by Alireza17224** [comment]
Same issue here

**Comment by ujithdon** [comment]
I tried many things, but still, the issue is there. Agree with @chramos

**Comment by lada** [comment]
The fix by @YasserShkeir works for me with SDK 49 (make sure you have expo-linear-gradient installed).
You can then create a patch with patch-package. Not perfect, but it works.
